src/libs/antares/study/parameters.cpp

```diff
@@ -1124,6 +1124,36 @@ static bool SGDIntLoadFamily_Legacy(Parameters& d,
         return true;
     }
 
+    if (key == "set-to-null-ntc-between-physical-out-for-first-step")
```

Comment:
We have a code duplication we did not have before with the use of **legacyLocalMatchingKeys**.
There may be a reason for this. Why ,